apps/ledger-live-desktop/src/live-common-setup-base.ts

```diff
@@ -1,11 +1,16 @@
-// @flow
 import "./env";
 import axios from "axios";
-
+import { setDeviceMode } from "@ledgerhq/live-common/hw/actions/app";
 import { listen as listenLogs } from "@ledgerhq/logs";
 import logger from "./logger";
 
-listenLogs(({ id, date, ...log }) => {
+// To let TS knows it can expect to find this global variable
+declare const __APP_VERSION__: string;
+
+// Sets the device actions implementation to "polling", and not "event"
+setDeviceMode("polling");
```

Comment:
If this passes QA, do we even need the multi implementation approach?

Reply:
yeah i was wondering if i should simplify everything, if we only keep the `polling` mechanism.

I would say: let's see if it passes QA first. If it does, we merge so we handle the 0x5515 in basic cases, and i open a new PR where i clean the `polling` / `event` . 

WDYT ?